Lib/glyphsLib/builder/constants.py

```diff
@@ -210,14 +210,14 @@
 # mode for the ufo2ft KernFeatureWriter whenever the GSFont contain a non-automatic
 # 'kern' feature.
 # See https://glyphsapp.com/tutorials/contextual-kerning
-# NOTE: Even though we use the default "skip" mode for the MarkFeatureWriter,
-# we still must include it this custom featureWriters list, as this is used
-# instead of the default ufo2ft list of feature writers.
-# This also means that if ufo2ft adds new writers to that default list, we
-# would need to update this accordingly... :-/
 DEFAULT_FEATURE_WRITERS = [
     {"class": "KernFeatureWriter", "options": {"mode": "append"}},
```

Comment:
also the mode="append" is no longer needed/used when building from .glyphs sources, because glyphsLib currently adds an insertion maker for ufo2ft to use in its default mode (called "skip" but actually checks for insertion markers). So you need to remove this from the KernFeatureWriter options, let it use the default KernFeatureWriter